scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -37,7 +37,6 @@
 
 import cnn_util
 import constants
-import data_utils
```

Comment:
Is there a reason we are removing this? can we not just comment it out like the others?